src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -169,16 +194,48 @@ export class CancerSummaryChart extends React.Component<CancerSummaryChartProps,
                                 }
                             </tbody>
                         </table>
+                        <If condition={this.props.showLinks}>
```

Comment:
we should inquire with product team as to whether these should open in new tab

Reply:
They do and they now do.